swupd/swupd.go

```diff
@@ -162,12 +162,20 @@ func (m Message) Process(printPrefix, line string) {
 			description = utils.Locale.Get("Resolving files that need to be installed")
 		case "download_packs":
 			description = utils.Locale.Get("Downloading required packs")
+		case "extract_packs":
+			description = utils.Locale.Get("Extracting required packs")
 		case "check_files_hash":
-			description = utils.Locale.Get("Verifying files integrity")
+			description = utils.Locale.Get("Verifying installed files")
```

Comment:
This description was updated to differentiate `check_files_hash` from `validate_fullfiles`. The `check_files_hash` step checks the hashes of files that are already installed on the target system. The `validate_fullfiles` step checks the hashes of fullfiles in the statedir that have not yet been installed to the target system.